source/plugins/pagespeed/examples.yml

```diff
@@ -30,3 +30,13 @@
     plugin_pagespeed_token: ${{ secrets.PAGESPEED_TOKEN }}
     plugin_pagespeed_url: https://lecoq.io
 
+- name: Succint report with PWA
+  uses: lowlighter/metrics@latest
+  with:
+    filename: metrics.plugin.pagespeed.svg
+    token: NOT_NEEDED
+    base: ""
+    plugin_pagespeed: yes
+    plugin_pagespeed_token: ${{ secrets.PAGESPEED_TOKEN }}
+    plugin_pagespeed_url: https://lecoq.io
+    plugin_pagespeed_pwa: yes
```

Comment:
To pass this test, you need to patch [`tests/mocks/api/axios/get/pagespeed.mjs`](https://github.com/lowlighter/metrics/blob/master/tests/mocks/api/axios/get/pagespeed.mjs) to integrate mocked data for PWA and make the builds pass

Reply:
done 😁